web/rainmaker/packages/employee/src/modules/employee/Inbox/index.js

```diff
@@ -60,7 +61,11 @@ class Inbox extends Component {
       return {
         labelName: obj.displayName,
         labelKey: `ACTION_TEST_${obj.displayName.toUpperCase().replace(/[._:-\s\/]/g, "_")}`,
-        link: () => setRoute(obj.navigationURL)
+        link: () =>{  
+          if (obj.displayName === "Apply TL" ){
```

Comment:
Please use url to check the condition instead of display name

Reply:
If I use URL to check condition this will trigger for not required actions for example search Tl does not require isRequiredDocument flag.